extensions/mssql/src/objectManagement/utils.ts

```diff
@@ -43,7 +43,16 @@ export function isValidSQLPassword(password: string, userName: string = 'sa'): b
 	return !containsUserName && password.length >= 8 && password.length <= 128 && (hasUpperCase + hasLowerCase + hasNumbers + hasNonAlphas >= 3);
 }
 
-// Converts number to two decimal placed string
+/*
+* Converts number to two decimal placed string
+*/
 export function convertNumToTwoDecimalStringinMB(value: number): string {
 	return localizedConstants.StringValueInMB(value?.toFixed(2));
 }
+
+/*
+* Converts string value coming from STS to pascal case, (first character to uppercase)
```

Comment:
pascal case isn't just the first character. Beginning of every word is uppercase, like PascalUpper

Reply:
Here we are already getting the Pascal case from the SMO, but typescript converts the first char to lowerCase, so named the method as PascalCase, based on the output, will update the comment clearly and please feel free to suggest a name here :)

Reply:
Updated the name as what we are doing in the method!